gamemodes/zombieplague/gamemode/extraitems/zp_antidote_bullets.lua

```diff
@@ -30,6 +31,9 @@ function ExtraItem:OnBuy(ply)
 		if !ShouldIgnore && Attacker == ply && !RoundManager:IsSpecialRound() then
 			if !RoundManager:LastZombie() then
 				InfectionManager:Cure(Target, Attacker)
+				Attacker:GiveAmmoPacks(cvars.Number("zp_ap_cure_zombie", 3))
```

Comment:
If we put this, it will double points, frags, etc

Reply:
I was wrong, we're not giving ammo packs for curing, but we're already giving points and frags